src/server/storage/projects.js

```diff
@@ -63,7 +63,7 @@
         }
 
         getId() {
-            return this._id;
+            return this._id.toString();
```

Comment:
are we sure that this ID isn't being used somewhere else directly relying on the fact that this is a valid `ObjectId`